### ISSUE TITLE
Add if device is root/non-root to exported patch logs

### ISSUE THREAD

**Issue by KobeW50** [ReVanced/revanced-manager#1405]
### Feature description

Add a line to exported patch logs which will say if a device is rooted or not.

### Motivation

It can be helpful when troubleshooting.

### Additional context

Here is an example of how it would look:

```
~ Device Info
ReVanced Manager: 1.13.1
Build: release
Model: SM-S906U1
Android version: 13
Supported architectures: arm64-v8a, armeabi-v7a, armeabi
Root: False

~ Patch Info
App: com.google.android.apps.youtube.music v6.21.51
...
```
Alternatively, the bool can be titled `Environment:`, and the responses can be either `Root` or `Non-root`. Here is what I mean:

```
~ Device Info
ReVanced Manager: 1.13.1
Build: release
Model: SM-S906U1
Android version: 13
Supported architectures: arm64-v8a, armeabi-v7a, armeabi
Environment: Non-root

~ Patch Info
App: com.google.android.apps.youtube.music v6.21.51
...
```

### Acknowledgements

- [X] This request is not a duplicate of an existing issue.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] The issue is solely related to the ReVanced Manager

**Comment by oSumAtrIX** [comment]
Should say `Root permissions: Yes/No`

**Comment by validcube** [comment]
🥞 The fix for this issue (#1407) has been released!
